### DISCUSSION TITLE
Typeorm vs Prisma: extremely slow request with Prisma comparing to typeorm.

### DISCUSSION THREAD

**Discussion by monapasan** [prisma/prisma#9552] [Q&A] [answered]
I am currently testing different ORMs to find a good fit for our project.
I write a very basic query with a nested write in Prisma which takes an extremly long time.

See my schema:
```
model Context {
  id          Int         @id @default(autoincrement())
  contextName String
  createdAt   DateTime    @default(now())
  statements  Statement[]
}

model Statement {
  id        Int      @id @default(autoincrement())
  content   String
  context   Context  @relation(fields: [contextId], references: [id], onDelete: Cascade)
  contextId Int
  createdAt DateTime @default(now())
}
```
I create a new context with around 600 statements inside.
This the code sample:
```
const context = await this.prisma.context.create({
      data: {
        contextName: contextData.name,
        statements: {
          create: statements,
        },
      },
      include: {
        statements: true,
      },
    });

```

I've tried to debug the request but haven't noticed anything suspicious from the Prisma output. 

Check out [the repo](https://github.com/monapasan/prisma-slow-request-to-remote-db) to see the full code and the request.

If I write **to the local DB** with prisma it takes around 4s vs **to remote DB** is more than 100s

The same operation with typeorm takes around  2s to the remote DB.

Any hints on how can improve the query?

**Comment by TasinIshmam** [comment, marked as answer]
Responded and resolved in #9612

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
